### PR TITLE
Create Contribution.md

### PR DESCRIPTION
**Pull request by Parth4git**
Solved issue #230 
kindly review my PR. @Quramy

### CODE REVIEW

File: Contribution.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
+# CSS Modules TypeScript Contribution Guidelines
```
Comment (Quramy) [inline Contribution.md:1]: Project name is not “CSS Modules TypeScript” but “typed-css-modules”

**Comment by Parth4git** [comment]
@Quramy  I have corrected it. now, you can review my PR.

**Comment by Parth4git** [comment]
I have corrected my mistake. So, you can review my PR

**Comment by Parth4git** [comment]
@Quramy Please response to my PR

**Review by Quramy — APPROVED** [review]

**Comment by Quramy** [comment]
@Parth4git Thanks!

**Pull request merged**